### ISSUE TITLE
0ur as a result of a glitch

### ISSUE THREAD

**Issue by myangelaku** [circleguard/circleguard#128]
https://youtu.be/B6BBsRGwk0A

don't really know what to call this

**Comment by tybug** [comment]
I can't reproduce. Exactly which one of the loadables did you use to load these two replays that resulted in 0 ur? Were they local replays?

**Comment by myangelaku** [comment]
> 
> 
> I can't reproduce. Exactly which one of the loadables did you use to load these two replays that resulted in 0 ur? Were they local replays?
> 
> If you still have the loadables you used in this run, a screenshot of those would be very helpful.

I'm pretty sure I used the user as a loadable

**Comment by tybug** [comment]
[he doesn't have that play in his top 100](https://osu.ppy.sh/users/5111479), so if you did use a user it must have been with a mod specified? I tried with `HDNC` and couldn't reproduce still.

**Comment by myangelaku** [comment]
> [he doesn't have that play in his top 100](https://osu.ppy.sh/users/5111479), so if you did use a user it must have been with a mod specified? I tried with `HDNC` and couldn't reproduce still.
> 
> ![image](https://user-images.githubusercontent.com/31628143/107862947-66573e80-6e1e-11eb-920c-50f6ba30a859.png)

I can assure you that I did use user loadable
Maybe it mixed something up and the cursor data is from another play?

**Comment by tybug** [comment]
yeah not sure, that's the best explanation but I've no idea how that could happen, something weird happened here for sure

**Comment by myangelaku** [comment]
It happened again, this time again while investigating a user!
![](https://myangel.aku.lu/7BSXck4z.png)

replay: https://osu.ppy.sh/scores/osu/3087396074
open in circleguard: circleguard://m=1400686&u=11083194&m1=HDDT
cvUR: 0.00 according to https://github.com/circleguard/circleguard

Theory: It could be related to the player/map before that loadable
